pallets/aggregate/src/lib.rs

```diff
@@ -340,9 +346,15 @@ pub mod pallet {
     /// Shortcut to get the Domain type from config.
     #[derive(Encode, Decode, TypeInfo, MaxEncodedLen)]
     #[scale_info(skip_type_params(T))]
-    pub(crate) struct Domain<T: Config>(DomainType<T>);
-
-    impl<T: Config> Domain<T> {
+    pub struct Domain<T: Config>(DomainType<T>);
```

Comment:
also here, don't make this struct public: this approach leads to a hight coupled code

Reply:
pub(crate)I don't need Domains in runtime anymore as we changed the payer account
undoing this